src/main/java/org/folio/innreach/util/JsonHelper.java

```diff
@@ -53,4 +58,17 @@ public <T> T fromJson(InputStream inputStream, Class<T> valueType) throws IOExce
     return obj;
   }
 
+
+  public static Long getCheckOutTimeDuration(List<ConfigurationDTO> configData) {
+    log.info("JsonHelper:getCheckOutTimeDuration started");
```

Comment:
Not sure what the two log info messages are doing in this method. Can you please explain?